src/parser/monk/mistweaver/modules/spells/RenewingMist.js

```diff
@@ -15,20 +18,31 @@ class RenewingMist extends Analyzer {
   gustsHealing = 0;
   lastCastTarget = null;
   healingHits = 0;
+  numberToCount = 0;
+
 
   on_byPlayer_cast(event) {
     const spellId = event.ability.guid;
+
     if (SPELLS.RENEWING_MIST.id !== spellId) {
       return;
+    } 
+    if (this.combatants.players[event.targetID]) {
+      if (this.combatants.players[event.targetID].hasBuff(SPELLS.ESSENCE_FONT_BUFF.id, event.timestamp, 0, 0) === true) {
+        this.numberToCount++;
```

Comment:
Another style item, but we have chosen to use `+= 1` to increment. So just rework this is `this.numberToCount += 1;`

Reply:
Will do Mr.BossMan